internal/cmd/argus/instance/create/create.go

```diff
@@ -102,12 +103,17 @@ func NewCmd(p *print.Printer) *cobra.Command {
 				s.Stop()
 			}
 
-			operationState := "Created"
-			if model.Async {
-				operationState = "Triggered creation of"
+			switch model.OutputFormat {
```

Comment:
Move the `switch case` inside the `outputResult` function for uniformity with the other commands

Reply:
same for all other commands

Reply:
shifted condition handling in the outputresult